kokoro/docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM docker.io/library/debian:bullseye AS build
+WORKDIR /app
+COPY . .
+RUN kokoro/rodete/fetch_dependencies.sh
+RUN rm -rf build
+RUN meson build
+RUN meson compile -C build
+RUN meson test --print-errorlogs -C build
+RUN meson install -C build
+
+FROM docker.io/library/debian:bullseye
+COPY --from=build /usr/local /usr/local
+COPY --from=build /app/kokoro/docker/glome-start /usr/local/sbin
```

Comment:
Any reason why this is not just `COPY kokoro/docker/glome-start /usr/local/sbin`?

Reply:
Done.